portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -86,6 +88,37 @@ definition {
 		}
 	}
 
+	@description = "LPS-143552. Verify that remote app of type Custom Element can be displayed by portal instance"
+	@priority = "5"
+	test CustomElementCanBeDisplayedByInstance {
+		property portal.acceptance = "true";
+		property test.assert.warning.exceptions = "true";
+		property test.name.skip.portal.instance = "RemoteApps#CustomElementCanBeDisplayedByInstance";
+
+		task ("Create virtual instance") {
+			SAML.addVirtualInstances(virtualHosts = "www.able.com");
```

Comment:
Use JSON instead for setup phase, there's HeadlessPortalInstanceAPI.addPortalInstance()